.github/workflows/release.yml

```diff
@@ -162,48 +162,10 @@ jobs:
       - name: Create bundle (macOS)
         run: |
           target=${{ matrix.build.target }}
-          BUNDLE_VERSION=$(grep -Po 'version = "\K.*?(?=")' -m 1 Cargo.toml)
```

Comment:
On MacOS the grep is not GNU grep like linux, but it's unix grep, so this command won't work correctly. That why I restored to the original one

Reply:
I see, thanks, always fun :upside_down_face: